h2o-core/src/main/java/water/rapids/ast/prims/reducers/AstCumMax.java

```diff
@@ -3,9 +3,7 @@
 
 public class AstCumMax extends AstCumu {
   @Override
-  public int nargs() {
-    return 1 + 1;
-  }
+  public int nargs() { return 1 + 2; }
```

Comment:
All these overrides are not necessary, since base `AstCumu` defines just the same number of args.
And in general when some class overrides `nargs()` it is also supposed to override `args()`...